mesonbuild/backend/vs2010backend.py

```diff
@@ -603,7 +648,11 @@ def gen_vcxproj(self, target, ofname, guid):
         # Prefix to use to access the source tree's root from the vcxproj dir
         proj_to_src_root = os.path.join(down, self.build_to_src)
         # Prefix to use to access the source tree's subdir from the vcxproj dir
-        proj_to_src_dir = os.path.join(proj_to_src_root, target.subdir)
+        #proj_to_src_dir = os.path.relpath(
```

Comment:
__[Flake8]__

[E265] block comment should start with '# '


<!-- Comment by [SideCI](https://sideci.com) -->